machines/bleShare/scan/scanMachine.ts

```diff
@@ -608,7 +608,7 @@ export const scanMachine =
             },
             rejected: {
               on: {
-                DISMISS: {
+                RETRY: {
```

Comment:
If this is changed to retry, should the button triggering this event be also changed or was this change intended for the other, i.e. receiveVC machine?

Reply:
When the shared vc is not stored into the storage then we will show the error status screen on both wallet and verifier and in wallet(in scanLayout.tsx) we are calling SharingStatusModal file for all types of errors and we are passing retry and goto home methods to this file and these methods are common for all error types so to support this i have changed dismiss to retry here.